opencti-platform/opencti-front/src/private/components/common/files/CustomFileUploader.tsx

```diff
@@ -60,12 +63,21 @@ const CustomFileUploader: FunctionComponent<CustomFileUploadProps> = ({
   label,
   acceptMimeTypes,
   sizeLimit = 0, // defaults to 0 = no limit
+  formikErrors,
 }) => {
   const { t_i18n } = useFormatter();
   const classes = useStyles();
   const [fileNameForDisplay, setFileNameForDisplay] = useState('');
   const [errorText, setErrorText] = useState('');
 
+  useEffect(() => {
+    if (formikErrors?.file) {
```

Comment:
By passing the entire `formikErrors` object and hardcoding the property `.file` here we force to use this component with the property file instead of leaving the liberty to the caller to chose by itself how the property should be named. But I see it is already made this way for `setFieldValue` below so... Ok, don't bother change the entire component.